.github/workflows/lint.yml

```diff
@@ -7,28 +7,16 @@ on:
       - 'docs'
 
 jobs:
-  clang-format:
-
-    runs-on: ubuntu-24.04 # latest
-
-    steps:
-    - name: Checkout Sources
-      uses: actions/checkout@v4
-
-    - name: clang-format lint
-      run: |
-        ./format-check.py
-
-  autopep8:
+  lint:
     runs-on: ubuntu-24.04 # latest
 
     steps:
     - name: Checkout Source
       uses: actions/checkout@v4
 
-    - name: Build and Test
+    - name: Format and Check Diff
       run: |
-        python3 -m pip install --upgrade setuptools
-        python3 -m pip install --upgrade wheel
         python3 -m pip install --upgrade autopep8
-        python3 -m autopep8 --exit-code --diff --recursive awscrt test .builder setup.py
+        python3 scripts/format-all.py
+        git diff
```

Comment:
do we want both?

Reply:
I switched this to just use the formatting script we already have. But that script just changes the code in-place, it doesn't have a "raise error if different" mode

`git diff --quiet` has exit code 1 if anything has changed, but it doesn't print anything

so I'm doing `git diff` before that, to show what's different (but it always exits with 0) before that

Reply:
I added comments to the file...